blackmarket/UI/Common/PaymentMethodsView.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  PaymentMethodsView.swift
+//  blackmarket
+//
+//  Created by Daniel Parra on 12/6/22.
+//
+
+import SwiftUI
+
+struct PaymentMethodsView: View {
+    var body: some View {
+        VStack(spacing: UI.PaymentMethodsView.VStack.spacing) {
+            Text("Payment methods")
+                .bold()
+                .font(.headline)
+            HStack(spacing: UI.PaymentMethodsView.HStack.spacing) {
+                VStack {
+                    Image("cardIcon")
```

Comment:
I think this should be a constant, first to be consistent with other UI resources, and secondly to avoid misspellings and repetitions. Please check other occurrences of Images in this PR.